### DISCUSSION TITLE
Combine into one

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7175] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

How do I combine multiple images into one without losing quality., both vertically and horizontally?

**Comment by snibgo** [comment]
You can put them side-by-side with `+append` or one below the other with `-append`. See https://imagemagick.org/script/command-line-options.php#append

For example:
```
magick in1.png in2.png +append out.png
```
Does that help?

**Reply by Jay123210599** [reply]
no, I mean pictures like these.
![ikkitousen_xtreme_xecutor_episode_8_snake_fight__by_mr40_dgvy1jq](https://github.com/ImageMagick/ImageMagick/assets/141977822/17e1c016-832d-44c8-af59-03c08337ca05)
![ikkitousen_stkr__hakufu_punished_for_the_bus_race_by_mr40_dgtbs4e](https://github.com/ImageMagick/ImageMagick/assets/141977822/aa4dbcea-2ed4-43e3-b72f-fb7d95c9fa32)
![ikkitousen_shugaku_toshi_keppu_roku_bus_race_1_by_mr40_dgtbogu](https://github.com/ImageMagick/ImageMagick/assets/141977822/3ac4d504-8189-4cb4-9260-915317bae8d3)
![ikkitousen_xtreme_xecutor_episode_3_too_much_tv_by_mr40_dgrekvf](https://github.com/ImageMagick/ImageMagick/assets/141977822/ea97227c-1c3c-4783-9c76-ad95ebc91434)

**Comment by snibgo** [comment]
In all of those examples, images are placed side-by-side, or beneath each other, or both. For example, the final example could be built like this (Windows BAT syntax):
```
magick ^
  ( in1.png in2.png +append +repage ) ^
  ( in3.png in4.png +append +repage ) ^
  -append +repage ^
  out.png
```
